src/cpp/ext/otel/otel_plugin.cc

```diff
@@ -202,5 +202,48 @@ absl::flat_hash_set<std::string> OpenTelemetryPluginBuilder::BaseMetrics() {
           OTelServerCallRcvdTotalCompressedMessageSizeInstrumentName())};
 }
 
+absl::string_view StatusCodeToString(grpc_status_code code) {
```

Comment:
We already have a function to do this:

https://github.com/grpc/grpc/blob/fb635a1959e8828a70194134c29a87c0c6b8c0fd/src/core/lib/channel/status_util.h#L37

Reply:
ah Thanks!